README.md

```diff
@@ -45,6 +45,8 @@ Click on the [CONTRIBUTING.md](https://github.com/amandamartin-dev/women-led-ope
 
 [Willow (GHOST)](https://github.com/ghostdevv) - [Many repositories](https://github.com/ghostdevv?tab=repositories)
 
+[Sue Spencer](https://github.com/slspencer) - [Seamly2D](https://github.com/fashionfreedom/seamly2d)
```

Comment:
Thank you for contributing! Do you mind placing this alphabetically to help with organization?  I"ll add that detail to the contributing guide.